app/views/admin/students/index.html.slim

```diff
@@ -14,7 +14,7 @@ table.table
     - @students.each do |student|
       tr data-id='#{student.id}'
         td = image_tag(avatar_for(student), {size:'100x100'})
-        td = student.full_name
+        td = link_to student.full_name, "http://#{ENV['LHL_CRM_HOST']}/admin/student_applications?converted=true&search=#{student.first_name}+#{student.last_name}", target: '_blank'
```

Comment:
Wouldn't their email address be a better search criteria? If it's nil then you could fall back